charter/2023-09-01.md

```diff
@@ -163,11 +163,10 @@ An affiliation MAY submit one ballot that ranks candidates in their preferred or
 
 #### Choosing a Chair
 
-* At any given time there may be up to three co-chairs, each holding one seat. Each seat defines a 2 year cycle of service.
-* In the first election after ratification of this charter, all seats will be up for election. Thereafter, in each year, a single election will be held to fill any vacant seats.
-* In the case of interim vacancy, the remaining chairs may appoint a co-chair for each open seat, hold an election for the same, or wait until the next election, at their discretion. If the chairs do not take any action, the seat will automatically be up for election in any cycle. Any such interim appointments or elections shall hold the seat until the end of its natural cycle.
+* At any given time there may be up to three co-chairs, each holding one seat for a 2 year cycle, or shorter in case they step down or are removed.
+* In the first election after ratification of this charter, all seats will be up for election. Thereafter, elections will be held whenever there are vacant seats.
 * Reelection is restricted to two consecutive terms, with the possibility of being reelected after sitting out one election cycle.
```

Comment:
This does not say anything about staggering elections.

There needs to be some mechanism by which to select one or two co-chairs who will initially serve a one or three year term. This would set up staggered elections going forward.

I think the intent is that elections are meant to be held in the same week of each year, except when filling seats that become vacant outside of this cycle. I think that the intent is also that co-chairs elected to fill such mid-term vacancies are meant to serve to the end of that term, not for two years from their election.

None of the above is expressed by the existing text, nor by my suggested changes below, which should be applied whether or not the above comments are acted upon.
```suggestion
* At any given time there may be up to three co-chairs, each holding one seat for a two year term, which may be shortened if they step down or are removed.
* In the first election after ratification of this charter, all seats will be up for election. Thereafter, elections will be held whenever there are vacant seats.
* Reelection is restricted to two consecutive terms, with the possibility of being reelected after sitting out one election cycle.
```